### PR TITLE
Update pokemon_pb.erl 

### PR DESCRIPTION
**Pull request by decoda**
use erlang:element/2 instead of tuple_to_list to get record name

### CODE REVIEW

**Comment by lukebakken** [comment]
Hello -

Thank you for taking the time to do this work. Do you have benchmarks or other evidence that these changes should be made? Thanks!

File: src/pokemon_pb.erl [hunk 1]
```diff
@@ -107,13 +107,13 @@ int_to_enum(_,Val) ->
     Val.
 
 %% DECODE
-decode_pikachu(Bytes) when is_binary(Bytes) ->
```
Comment (lukebakken) [inline src/pokemon_pb.erl]: This seems like an unnecessary change. @javajolt @alexmoore @tburghart - care to weigh in?

File: src/pokemon_pb.erl [hunk 2]
```diff
@@ -87,7 +87,7 @@ pack(FNum, repeated_packed, Data, Type, _) ->
     protobuffs:encode_packed(FNum, Data, Type);
 
 pack(FNum, _, Data, _, _) when is_tuple(Data) ->
-    [RecName|_] = tuple_to_list(Data),
+    RecName = erlang:element(1, Data),
```
Comment (lukebakken) [inline src/pokemon_pb.erl]: This seems like a good change.

@javajolt @alexmoore @tburghart - care to weigh in?

File: src/pokemon_pb.erl (hunk 1 above)
Comment (alexmoore) [inline src/pokemon_pb.erl]: @lehoff Do you know of any authoritative Erlang best practice here?

File: src/pokemon_pb.erl (hunk 2 above)
Comment (javajolt) [inline src/pokemon_pb.erl]: Agreed
